src/instruments/src/EFB/ATC/ATC.scss

```diff
@@ -0,0 +1,49 @@
+.atc-buttons {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+}
+
+.atc-button {
+    margin: 5px;
```

Comment:
Same. Use of custom cases and sizes should be strictly minified for UI consistency. 

https://tailwindcss.com/docs/margin

`@apply m-x active:text-yellow-500;`